web/app/themes/wordpress-scaffold/lib/Grrr/PostTypes/PostTypeStub.php

```diff
@@ -0,0 +1,11 @@
+<?php namespace Grrr\PostTypes;
+
+use Timber;
+
+abstract class PostTypeStub {
```

Comment:
Just stumbled on the issue that the native post types don't extend the `PostTypeAbstract`, but we were assuming that all post types support the `get_structured_data` method. Could also be a trait, or a more complex construction.

Thoughts?

Reply:
Het voelt toch meer dat dit een trait moet zijn.